index.js

```diff
@@ -1,21 +1,30 @@
-import isUndefined from 'lodash/lang/isUndefined';
+'use strict';
+
+var isUndefined = require('lodash.isundefined');
 
 /**
  * Recursive dispatch, returns a function which iterates a series of commands
  * looking for one to handle the target and return a value. The commands adhere
  * to an interface of (target, fn) where the returned dispatch fn is passed
  * along to each command, where it can be used by the command. If the target
- * cannot be handled by a command the command returns undefined.
+ * cannot be handled by a command the command returns undefined. If none of the
+ * supplied commands handle the target, an error is thrown.
  *
  * @param commands
  * @returns {Function}
  */
-function dispatch(...commands) {
+
+module.exports = function dispatch() {
+  var commands = new Array(arguments.length);
```

Comment:
Nope, I have not looked at that doc. Where is it? Sounds like something the javascript language and v8 needs to fix. Using `Array.prototype.slice.call(arguments);` is used so broadly in all javascript everywhere, and to me is the most straight forward way to handle a variable number of arguments.